Components/App.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { ChangeEvent, useState, FC } from 'react';
+import { ITask } from './interfaces/ITask';
+import TodoTask from './ToDoTask/ToDoTask';
+import CustomDiv from './CustomDiv/CustomDiv';
+import CustomButton from './CustomButton/CustomButton';
+import CustomInput from './CustomInput/CustomInput';
+
+const App: FC = () => {
+  const [task, setTask] = useState<string>("");
+  const [deadline, setDeadline] = useState<number>(0);
+  const [todoList, setTodoList] = useState<ITask[]>([]);
+
+  const handleChange = (event: ChangeEvent<HTMLInputElement>): void => {
+    if (event.target.name === "task") {
+      setTask(event.target.value);
+    } else {
+      setDeadline(Number(event.target.value));
+    }
+  };
+
+  const addTask = (): void => {
+    const newTask = { taskName: task, deadline: deadline };
+    setTodoList([...todoList, newTask]);
+    setTask("");
+    setDeadline(0);
+  };
+
+  const completeTask = (taskNameToDelete: string): void => {
+    setTodoList(todoList.filter((task) => {
+      return task.taskName !== taskNameToDelete;
+    }));
+  };
+
+ //const CustomDiv: React.FC<CustomeDivProps> = ({ children, style , width , height , display , alignItems , justifyContent  }) => {
+  return (
+    <CustomDiv width="100%" height="150%"  display="flex"  justifyContent="center" alignItems="center"  style={{ backgroundColor: '#f0f0f0'}}>
+      <CustomDiv display="flex" justifyContent="center" alignItems="center" width="" height=""
```

Comment:
always it is display "flex". so why we pass it as a prop?

Reply:
when I am removing display flex it is giving error that's why